nixos/modules/services/web-apps/rss-bridge.nix

```
@@ -5,10 +5,18 @@ let
 
   poolName = "rss-bridge";
 
-  whitelist = pkgs.writeText "rss-bridge_whitelist.txt"
-    (concatStringsSep "\n" cfg.whitelist);
+  configAttr = { FileCache.path = "${cfg.dataDir}/cache/"; } // cfg.config;
```

```suggestion
  configAttr = lib.recursiveUpdate { FileCache.path = "${cfg.dataDir}/cache/"; } cfg.config;
```
otherwise specifying just FileCache will delete the path